### ISSUE TITLE
Pre-commit hook fails with `unrecognized arguments: --root`

### ISSUE THREAD

**Issue by browniebroke** [gauge-sh/tach#71]
Here is my pre-commit config file

```yaml
default_language_version:
  python: python3.12
repos:
  # Check dependencies between apps
  - repo: https://github.com/Never-Over/tach
    rev: "v0.2.0"
    hooks:
      - id: tach
```

When I try to commit, I get:

```sh
tach.....................................................................Failed
- hook id: tach
- exit code: 2
usage: tach [-h] {pkg,check,install,sync,clean} ...
tach: error: unrecognized arguments: --root
```

Seems like it was added in https://github.com/gauge-sh/tach/pull/62

I don't see any mention of the `--root` option in the help of tach check:

```sh
 ➜ tach check --help       
usage: tach check [-h] [--strict] [-e file_or_path,...]

Check existing boundaries against your dependencies and package interfaces

options:
  -h, --help            show this help message and exit
  --strict              Raise errors if any dependency constraints are unused.
  -e file_or_path,..., --exclude file_or_path,...
                        Comma separated path list to exclude. tests/, ci/, etc.
```

What am I missing?

**Comment by emdoyle** [comment]
Thanks for raising this issue! This feature was broken when using the pre-commit framework, sorry for the inconvenience. We just published a new version `0.2.1` which addresses this! You can check out #66 for more details.

If you adjust the `rev` in your posted config to `v0.2.1`, your config should work as-is.

**Comment by browniebroke** [comment]
Amazing, that worked! Thanks 🎉
